test/issues/99_canonical_with_junction_point.cpp

```diff
@@ -0,0 +1,61 @@
+
```

Comment:
License header missing.

Also, please add a short description of what this test is testing.